great_expectations/datasource/fluent/alloy_datasource.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import Literal, Union
+
+from great_expectations._docs_decorators import public_api
+from great_expectations.compatibility.pydantic import PostgresDsn
+from great_expectations.datasource.fluent.config_str import ConfigStr
+from great_expectations.datasource.fluent.sql_datasource import SQLDatasource
+
+
+@public_api
+class AlloyDatasource(SQLDatasource):
+    """Adds an alloy datasource to the data context.
+
+    Args:
+        name: The name of this alloy datasource.
+        connection_string: The SQLAlchemy connection string used to connect to the postgres database.
```

Comment:
Why do we call these `SQLAlchemy connection string`? We should remove SQLAlchemy.